src/components/NotesMindMap/NotesMindMapReducer.js

```diff
@@ -140,6 +139,9 @@ export const notesMindMapReducer = (state = defaultState, { type, data }) => {
     const clonedState = { ...state };
```

Comment:
We don't have a tree anymore, so I guess we should remove the `cloneTreeInState` function?

One of the reasons for the refactoring was to get rid of this function and make reducers simpler.

Reply:
Removed